api/common/configentry_webhook_test.go

```diff
@@ -112,7 +114,9 @@ func TestValidateConfigEntry(t *testing.T) {
 				lister,
 				c.newResource,
 				c.enableNamespaces,
-				c.nsMirroring)
+				c.nsMirroring,
+				c.consulDestinationNS,
+				c.nsMirroringPrefix)
 			require.Equal(t, c.expAllow, response.Allowed)
```

Comment:
Do we need to add an assertion here that the defaulting has worked (maybe check that the `DefaultNamespaceFields`  method was called on the mock config entry)?

Reply:
Added a test for the same. It is fairly primitive but let me know if that works of if you'd like something a little more fleshed out.

Reply:
that works, thanks!